CarpetLib/src/interpolate_3d_5tl.cc

```diff
@@ -120,11 +120,10 @@ void interpolate_3d_5tl(T const *restrict const src1, CCTK_REAL const t1,
   RT const s5fac = (t - t1) * (t - t2) * (t - t3) * (t - t4) /
                    ((t5 - t1) * (t5 - t2) * (t5 - t3) * (t5 - t4));
 
-// Loop over region
-#pragma omp parallel
+  // Loop over region
+  // #pragma omp parallel
```

Comment:
this should not be committed in this form. Rather the whole pragma should be removed.